lib/encryption.rb

```diff
@@ -0,0 +1,41 @@
+class Encryption
+  attr_reader :character_set
+
+  def initialize
+    @character_set = ("a".."z").to_a << " "
+  end
+
+  def indexed_alphabet
```

Comment:
Hash the alphabet to make it easier to use!!! Key for finding the characters that are assigned to certain indexes.